src/App.js

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from 'react';
+import { Button } from './components/Button/Button';
+
+function App() {
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
+    getData();
+  }, []);
+
+  const getData = async () => {
+    const response = await fetch('https://jsonplaceholder.typicode.com/users');
+    const data = await response.json();
+
+    getAllAvatars(data);
+  };
+
+  const getAllAvatars = async (data) => {
+    let newUsersList = [];
+
+    for (let i = 0; i < data.length; i++) {
```

Comment:
use `for await` instead this.
https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Statements/for-await...of

Reply:
@CarrioliDante Aca tienes todo.